app/src/main/java/com/ballboycorp/com/topik/SonarBugSample.kt

```diff
@@ -0,0 +1,19 @@
+package com.ballboycorp.com.topik
+
+interface Sample {
+    fun method()
+
+    fun anotherMethod()
+}
+
+class SampleImpl : Sample {
+    override fun method() {
+
+    }
+
+    override fun anotherMethod() {
+        var some = 1
+        some = some
```

Comment:
Remove or correct this useless self-assignment. [Details](https://fb41-221-141-140-219.jp.ngrok.io/project/issues?id=musooff_Topik_AYB-ZFlS8-xhCYxQvxYf&issues=AYB-q2G40n16_HoDUh6w&open=AYB-q2G40n16_HoDUh6w)